content/jmccann/hipchat/index.md

```diff
@@ -50,12 +50,18 @@ The Hipchat plugin supports reading credentials from the Drone secret store. Thi
 ```diff
 pipeline:
   slack:
-    image: plugins/hipchat
+    image: jmccann/drone-hipchat
     room: my-room
 -   auth_token: my-auth-token
 ```
 
-The above `auth_token` Yaml attribute can be replaced with the `HIPCHAT_AUTH_TOKEN` secret environment variable. Please see the Drone documentation to learn more about secrets.
+The above `auth_token` Yaml attribute can be replaced with the `HIPCHAT_AUTH_TOKEN` secret environment variable.
+Please see the [Drone documentation](http://readme.drone.io/0.5/secrets-with-plugins/) to learn more about secrets.
```

Comment:
we should probably use a hugo [short code](https://gohugo.io/extras/shortcodes/) for dynamically injecting the link to secrets. That way if the link changes (it has in my local branch) we can update in a single place.

I'm thinking the short code could look like this in the markdown file:

```
{{< secret-link >}}Drone Documentation{{</ secret-link >}}
```

and then `/layouts/shortcodes/secret-link.html` could look like this:

```
<a href="http://readme.drone.io/0.5/secrets-with-plugins/">
{{ .Inner }}
</a>
```

I'm not sure whether we should use `.Inner` or if we should pass a parameter to the shortcode like this `{{ < secret-link "Drone Documentation" > }}`. Thoughts?